languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -34110,6 +34124,42 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_TYPOGRAPHIE" name="Typographie" type="typographical">
+     <rulegroup id="VIRGULE_TEMPS" name="virgule temps">
+      <rule>
+          <pattern>
+              <unify>
+                  <feature id="person"/>
+                  <feature id="tense"/>
+                  <marker>
+                      <token postag="R.* pers.* suj.*" postag_regexp="yes"/>
+                      <token postag="V.*" postag_regexp="yes"/>
+                      <token postag="R.* pers.* suj.*" postag_regexp="yes"/>
+                      <token postag="V.*" postag_regexp="yes"/>
```

Comment:
This is one of the reasons why it doesn't work: the unification tries to find tense in the pronoun, which it can't. You would have to unify-ignore the pronouns.